executions/graphql-kotlin-dataloader/README.md

```diff
@@ -0,0 +1,68 @@
+# GraphQL Kotlin Data Loader
+[![Maven Central](https://img.shields.io/maven-central/v/com.expediagroup/graphql-kotlin-dataloader.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.expediagroup%22%20AND%20a:%22graphql-kotlin-dataloader%22)
+[![Javadocs](https://img.shields.io/maven-central/v/com.expediagroup/graphql-kotlin-dataloader.svg?label=javadoc&colorB=brightgreen)](https://www.javadoc.io/doc/com.expediagroup/graphql-kotlin-dataloader)
+
+Data Loaders are a popular caching pattern from the [JavaScript GraphQL implementation](https://github.com/graphql/dataloader).
+
+Since `graphql-java` provides [support for this pattern](https://www.graphql-java.com/documentation/v16/batching/)
```

Comment:
should we link to the v18 instead? https://www.graphql-java.com/documentation/batching